packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -1012,18 +989,21 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/val
  emitAuditEvents: false
  image:
 -  repository: openpolicyagent/gatekeeper
--  release: v3.1.0-rc.1
-+  repository: rancher/opa-gatekeeper
-+  tag: v3.1.0-rc.1
+-  release: v3.1.0
++  repository: rancher/openpolicyagent-gatekeeper
++  tag: v3.1.0
    pullPolicy: IfNotPresent
  nodeSelector: { kubernetes.io/os: linux }
  affinity: {}
-@@ -23,3 +23,8 @@
+@@ -23,3 +23,11 @@
    requests:
      cpu: 100m
      memory: 256Mi
 +global:
-+  systemDefaultRegistry: ""
++  cattle:
++    systemDefaultRegistry: ""
++    clusterId: ""
++    clusterName: ""
 +  kubectl:
 +    repository: rancher/istio-kubectl
```

Comment:
There is a standard rancher/kubectl, don't use the istio one.

Reply:
@cbron Would using the standard one cause any issues that you know of? This is what the chart was using since before it was converted to a dev chart, so I just kept it as is.

Reply:
I think it didn't exist. We can just leave for now.